.envrc

```diff
@@ -1 +1,3 @@
+source_up_if_exists
```

Comment:
I'd like to load environment variables from an `.envrc` in the root of all my Zoo projects. "if exists" makes this backwards compatible.